vendor/src/github.com/microsoft/hcsshim/layerutils.go

```diff
@@ -84,9 +85,14 @@ func layerPathsToDescriptors(parentLayerPaths []string) ([]WC_LAYER_DESCRIPTOR,
 	var layers []WC_LAYER_DESCRIPTOR
 
 	for i := 0; i < len(parentLayerPaths); i++ {
-		// Create a layer descriptor, using the folder path
+		// Create a layer descriptor, using the folder name
```

Comment:
Using the folder name matches the behavior that the in-box container infrastructure uses. This ensures that all clients of Windows Server Containers use the same string for generating the unique ID for shared base images. Without this change, docker containers and non-docker containers wouldn't be able to run at the same time.